rules/linux/persistence_shell_activity_via_web_server.toml

```diff
@@ -0,0 +1,111 @@
+[metadata]
+creation_date = "2023/03/04"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/03/04"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies suspicious commands executed via a web server, which may suggest a vulnerability and remote shell access."
+false_positives = [
+    """
+    Network monitoring or management products may have a web server component that runs shell commands as part of normal
+    behavior.
+    """,
+]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
```

Comment:
We need to test this one to see if it is Endgame compatible, if not, we should remove the Endgame tag

Reply:
Also, would be interesting to test this with Auditbeat too, and remove or add setup instructions as needed